api/db/migrations/001_initial_schema.down.sql

```diff
@@ -0,0 +1 @@
+-- TODO
```

Comment:
what's the difference between `.down.sql` and `.up.sql`

Reply:
I'm assuming how to rollback and how to update.

Reply:
Yep. Normally you only need to run up migrations but you need down ones to revert if you screwed up. I can teach you guys more about migrations in a meeting if you want.

Reply:
yes please haha, I wanted to learn more backend stuff